Library/Homebrew/extend/pathname.rb

```diff
@@ -490,6 +490,12 @@ def zipinfo
                  .encode(Encoding::UTF_8, invalid: :replace)
                  .split("\n")
   end
+
+  sig { params(parent: Pathname).returns(T::Boolean) }
+  def child_of?(parent)
```

Comment:
Can we avoid adding more monkey patches to `Pathname` in favour of e.g. a new `Utils` method? CC @dduugg for thoughts.